src/cli/tc.rs

```diff
@@ -111,3 +114,19 @@ fn display_config(config: &Config, n_plus_one_queries: bool) {
   let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
   Fmt::display(Fmt::table(seq));
 }
+
+fn write_output_file(filename: &String, config: &Config) -> Result<()> {
+  let filetype = Source::detect(filename)?;
+
+  let contents = match filetype {
+    Source::GraphQL => config.to_sdl(),
+    Source::Json => config.to_json().unwrap_or_default(),
+    Source::Yml => config.to_yaml().unwrap_or_default(),
+  };
+
+  Fmt::display(Fmt::success(
+    &format!("Schema has been written to {}", filename).to_string(),
+  ));
+  fs::write(filename, contents).unwrap();
```

Comment:
```suggestion
  fs::write(filename, contents).unwrap();
  Fmt::display(Fmt::success(
    &format!("Schema has been written to {}", filename).to_string(),
  ));
```

Reply:
Ah, yes. Missed the causality😅. Fixing it. Thanks.